Makefile

```diff
@@ -87,13 +87,14 @@ go.test.unit: $(KUBEBUILDER)
 # Generate manifests e.g. CRD, RBAC etc.
 manifests: vendor
 	@$(INFO) Generating CRD manifests
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/cache/... output:dir=cluster/charts/crossplane/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/compute/... output:dir=cluster/charts/crossplane/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/core/... output:dir=cluster/charts/crossplane/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/database/... output:dir=cluster/charts/crossplane/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/storage/... output:dir=cluster/charts/crossplane/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/workloads/... output:dir=cluster/charts/crossplane/crds
-	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:maxDescLen=0,trivialVersions=true paths=./apis/stacks/... output:dir=cluster/charts/crossplane/crds
+	rm -rf cluster/charts/crossplane/templates/crds
+	go run vendor/sigs.k8s.io/controller-tools/cmd/controller-gen/main.go crd:trivialVersions=true paths=./apis/cache/... output:dir=cluster/charts/crossplane/templates/crds
```

Comment:
I looked in the helm migration issue https://github.com/helm/helm/issues/5892 quickly to see where it talked about this requirement to move the CRDs to under the templates dir, but I couldn't find it quickly.  Can you help point me to it?

Reply:
I dug more and added more context about this to the description of the PR.

One can possibly add an if/else statement in `crds.yaml` to decide whether to copy crds folder. However, I couldn't find a good way to get helm version in the yaml. The closest is `{{if not Capabilities.TillerVersion }}` but that results in error in Helm3 saying `Error: parse error at (crossplane/templates/crds.yaml:1): function "Capabilities" not defined`

Reply:
Cool, from the PR description you added, it sounds like a good solution currently that works for both helm 2 and helm 3 is to just use the templates folder.  Thanks for clarifying all that!